ios/ios_project_setup.md

```diff
@@ -0,0 +1,61 @@
+# How To Create A New Project
+## Create Xcode project and Github repo
+1. Create repo on Github with `.gitignore` and `README.md`. Usually, repos should be named with the format `<project-name>-ios` e.g `bose-stetson-ios`
+
+1. Create Xcode Project
+
+1. In the Xcode project directory, run:
+    ```
+    git init
+    git remote add origin <remote-url>
+    git pull origin master
+    git push --set-upstream origin master
+    ```
+    > Pull before pushing to get the Github-generated `.gitignore` so ignored files don't end up in the repo
+
+1. Update `README.md` to use [Intrepid README template](https://github.com/IntrepidPursuits/sherpa/blob/master/readme-template)
+1. Add and commit, then push to `master`
```

Comment:
Do we have any guidelines on initial branches to create? ex: adding a `develop` branch